tapeio.c

```diff
@@ -424,6 +428,7 @@ int getrec (tape_handle_t mtape, void *buf, int len)
 {
   unsigned char byte [4];		/* 32 bits for length field(s) */
   unsigned long l;		/* at least 32 bits */
+  int i;
```

Comment:
Both functions (response and read) return and in....I don't think it matters that much.